tests/conftest.py

```diff
@@ -17,9 +17,21 @@
 
 import cript
 
+
+def _get_cript_tests_env() -> bool:
+    """
+    A little helper function, that ensure to mostly default to True.
+    Except if an explicit spelling of False is detected.
+    """
+    env_var = os.getenv("CRIPT_TESTS")
+    if env_var in ("false", "FALSE", "False"):
+        return False
+    return True
+
+
 # flip integration tests ON or OFF with this boolean
 # automatically gets value env vars to run integration tests
-HAS_INTEGRATION_TESTS_ENABLED: bool = os.getenv("CRIPT_TESTS").title() == "True"
+HAS_INTEGRATION_TESTS_ENABLED: bool = _get_cript_tests_env()
```

Comment:
why was this change necessary? the previous code functioned correctly and was clean

Reply:
It was necessary, because the old version didn't tolerate no specification of the ENV variable.

That case `os.getenv("CRIPT_TESTS")` returns `None`.
And you can't call `.title()` on a `None` object.

Reply:
can't you just add a small try except block then?

Reply:
an except block would work too, but I don't like the logic of an except block here.

I don't really care though.
This is a tiny detail.